courses/fundamentals_of_ada/160_genericity.rst

```diff
@@ -449,16 +449,14 @@ Quiz
 
  .. container:: column
 
-  .. container:: latex_environment scriptsize
-
-   What is printed when :ada:`Instance` is called?
+   What is printed when we call :ada:`Instance (Object => 0);`?
```

Comment:
Instance (Object => 0); causes a compilation error, I think the fix is incomplete

Reply:
Turning that into an executable quiz would help clarify the matter.